charts/spire/charts/spire-server/templates/_helpers.tpl

```diff
@@ -217,3 +217,11 @@ The code below determines what connection type should be used.
 {{-   end }}
 {{ $args | toYaml }}
 {{- end -}}
+
+{{- define "spire-server.controller-manager-class-name" -}}
```

Comment:
Looking at this wouldn't instanceName be a better name?

Assuming this represents an instance of a spire-server.

Reply:
Following the established k8s pattern. className is very common and I don't think we should deviate from it.

Reply:
Do you have a reference to that common pattern of className usage? I really have trouble understanding this naming. Hope I can read up somewhere a bit to understand this naming convention.

Reply:
Example of storage class useage:
https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/#using-dynamic-provisioning

Example of ingress class usage:
https://kubernetes.io/docs/concepts/services-networking/ingress/#the-ingress-resource

StorageClass details:
https://kubernetes.io/docs/concepts/storage/storage-classes/

Ingress api documentation around ingress class:
https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-class

Ingress-nginx charts configurables for classes:
https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/values.yaml#L76-L81
https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/values.yaml#L108-L125